mmv1/products/datafusion/Instance.yaml

```diff
@@ -237,16 +249,54 @@ properties:
         description: |
           The IP range in CIDR notation to use for the managed Data Fusion instance
           nodes. This range must not overlap with any other ranges used in the Data Fusion instance network.
-        required: true
+        required: false
         immutable: true
       - !ruby/object:Api::Type::String
         name: 'network'
         description: |
           Name of the network in the project with which the tenant project
           will be peered for executing pipelines. In case of shared VPC where the network resides in another host
           project the network should specified in the form of projects/{host-project-id}/global/networks/{network}
-        required: true
+        required: false
         immutable: true
+      - !ruby/object:Api::Type::Enum
+        name: 'connectionType'
+        description: |
+          Optional. Type of connection for establishing private IP connectivity between the Data Fusion customer project VPC and
+          the corresponding tenant project from a predefined list of available connection modes.
+          If this field is unspecified for a private instance, VPC peering is used.
+        values:
+          - :CONNECTION_TYPE_UNSPECIFIED
```

Comment:
maybe we can remove `CONNECTION_TYPE_UNSPECIFIED`.

Reply:
Yeah, I'd recommend removing that value!